src/ORM/Association.php

```diff
@@ -438,6 +438,15 @@ public function property($name = null)
         if ($name === null && !$this->_propertyName) {
```

Comment:
Similar code exist in [BelongTo](https://github.com/cakephp/cakephp/blob/d5c39cc4440c6a11fab3e245db4d967a8fa1602f/src/ORM/Association/BelongsTo.php#L87) and [HasOne](https://github.com/cakephp/cakephp/blob/d5c39cc4440c6a11fab3e245db4d967a8fa1602f/src/ORM/Association/HasOne.php#L72).